### ISSUE TITLE
Disambiguate the permalinks so that they do not all have the same programmatic name

### ISSUE THREAD

**Issue by ChrisXKeroack** [intel/openfl#851]
**Describe the bug**
Because all permalinks have the same programmatic name, It will take extra time for e.g., screenreader user to get context on which permalink goes to what, and lack of unique / short name will make working with these links awkward for e.g., voice input

**To Reproduce**
1. start a screenreader
2. https://openfl.readthedocs.io/en/latest/index.html
3. navigate through interactive elements in the page content

notice: all permalinks are described the same

**Expected behavior**
Each permalink should also include the text of the heading next to it, so that when a screenreader e.g. uses common feature of screenreader to get a list of all links, the link names will all be unique

**Screenshots**
![image](https://github.com/securefederatedai/openfl/assets/104644211/1131ae92-9896-4bbb-b272-968b1e78c723)

**Desktop:**
This issue is not OS or browser specific, but for tracking purposes, testing was done on Win11, in Edge Version 114.0.1823.58 (Official build) (64-bit), and the screenreader in use was the common 3rd party screenreader NVDA (http://www.nvaccess.org).

**Smartphone:**
This issue is not device, OS, or browser specific.

**Additional context**
This is an accessibility issue, please annotate this bug with the 'accessibility' label.

**Comment by fstrr** [comment]
This is an issue with the [Read The Docs platform](https://github.com/readthedocs/readthedocs.org).

**Comment by theakshaypant** [comment]
Closing. As mentioned by @fstrr, this is an issue with [Read The Docs platform](https://github.com/readthedocs/readthedocs.org).
